paragon/_variables.scss

```diff
@@ -17,3 +17,11 @@ $link: $blue;
 $link-color: $blue;
 $inline-link-color: $blue;
 $info: $light;
+
+
+$font-family-sans-serif: $avenir;
+
+p > a[href]:not(.btn), 
+a.inline-link {
+    font-weight: 500;
+}
```

Comment:
I wonder if this should be in `_overrides.scss` since it isn't a variable. Also, let's use `$font-weight-semi-bold` as the value instead of `500`.